app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -73,7 +79,11 @@ const dynamicTags = computed<Tag[]>(() => {
   }))
 })
 
-const tags = computed(() => (dynamicTags.value.length > 0 ? dynamicTags.value : staticTags.value))
+const customTags = computed(() => props.input[CustomDropdownItemsKey]?.map(tagFromCustomItem) ?? [])
+const tags = computed(() => {
+  const standardTags = dynamicTags.value.length > 0 ? dynamicTags.value : staticTags.value
+  return customTags.value.concat(standardTags)
```

Comment:
There's nothing wrong with `concat` really, but so far we tend to use array spreads for this. That makes is explicit that `standardTags` is in fact an array (as `concat` also behaves like `push` for other types).

```suggestion
  return [...customTags.value, ...standardTags]
```

Reply:
Done